src/redux/books/books.js

```diff
@@ -1,13 +1,40 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
 const ADD_BOOK = 'books/books/ADD_BOOK';
 const REMOVE_BOOK = 'books/books/REMOVE_BOOK';
-export const addBook = (data) => (dispatch) => {
+const SHOW_BOOK = 'books/books/SHOW_BOOK';
+const baseApi = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/gPzOMZksyeRmN5CvYQTV/books';
```

Comment:
# Congratulations 🥇 
- Connect React-Redux application to an existing API
-  Create and remove books in a remote server